CSharpJobTemplate.yml

```diff
@@ -0,0 +1,72 @@
+# JOB LEVEL TEMPLATE:
+# Used to build CSharp
+# Reusable
+# Meant to be run from the single CSharpTemplate pipeline (default), or
+#   from a multi-pipeline such as publishing (should specify alternate params)
+
+parameters:
+- name: vmImage
+  displayName: vmImage
+  type: string
+  default: 'windows-latest'
+- name: ApiSpecSource
+  displayName: ApiSpecSource
+  type: string
+  default: -apiSpecGitUrl https://raw.githubusercontent.com/PlayFab/API_Specs/master/
+- name: CommitMessage
+  displayName: CommitMessage
+  type: string
+  default: Automated build from ADO Pipeline
+- name: GitDestBranch
+  displayName: GitDestBranch
+  type: string
+  default: doNotCommit
+- name: GitJSetupBranch
+  displayName: GitJSetupBranch
+  type: string
+  default: master
+- name: GitSdkGenBranch
+  displayName: GitSdkGenBranch
+  type: string
+  default: master
+- name: isVersioned
+  displayName: isVersioned
+  type: boolean
+  default: false
+- name: TestCompile
+  displayName: TestCompile
+  type: boolean
+  default: true
+- name: TestWin32NetCore
+  displayName: TestWin32NetCore
+  type: boolean
+  default: true
+- name: TestWin32Basic
+  displayName: TestWin32Basic
+  type: boolean
+  default: true
+- name: SelfTemplateResource
+  displayName: SelfTemplateResource
+  type: string
+  default: self
```

Comment:
SelfTemplateResource is unused in the other template, so it can be removed here too.